contracts/Sal.sol

```diff
@@ -20,17 +21,22 @@ contract allemp{
     function addemp (
         string memory _FirstName,
         string memory _LastName,
-        string memory _wallet_ddress,
+        string memory _wallet_address,

```

Comment:
Why wallet address is of type String it should be of type address 
Eg something like "address _wallet_address" to ensure that only valid Ethereum addresses can be used as employee wallet addresses.

Reply:
ok, I will update to Variable Type String For only Valid Ethereum Address.